### PR TITLE
update to blueocean-display-url 0.5.1

### PR DESCRIPTION
**Pull request by i386**
# Description

Now the UC has propagated its now safe to bump this.

# Submitter checklist
- [ ] Link to JIRA ticket in description, if appropriate.
- [ ] Change is code complete and matches issue description
- [ ] Appropriate unit or acceptance tests or explanation to why this change has no tests
- [ ] Reviewer's manual test instructions provided in PR description. See Reviewer's first task below.
- [ ] Ran Acceptance Test Harness against PR changes.

# Reviewer checklist
- [ ] Run the changes and verified the change matches the issue description
- [ ] Reviewed the code
- [ ] Verified that the appropriate tests have been written or valid explanation given

@reviewbybees

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -208,6 +208,12 @@
             <version>${project.version}</version>
         </dependency>
 
+        <dependency>
```
Comment (i386) [inline pom.xml]: Was not declared in the root POM, fixed.

**Comment by michaelneale** [comment]
@i386 can you update this to master?

**Comment by i386** [comment]
Done

**Comment by michaelneale** [comment]
:bee:

**Pull request merged**